www/src/pages/components/checkbox/react/index.mdx

```diff
@@ -114,42 +74,24 @@ class CheckboxExample extends React.Component {
 [Indeterminate checkboxes](https://css-tricks.com/indeterminate-checkboxes/) are used when not all items in a field are selected.
 
 ```jsx
-class CheckboxExample extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isChecked: undefined,
-        };
-
-        this.onChange = this.onChange.bind(this);
-    }
-
-    onChange(isChecked) {
-        this.setState({
-            isChecked,
-        });
-    }
-
-    render() {
-        const { isChecked } = this.state;
-
-        return (
-            <Checkbox
-                isIndeterminate={isChecked === undefined}
-                isChecked={isChecked}
-                onChange={this.onChange}
-            >
-                Select all cities
-            </Checkbox>
-        );
-    }
+function CheckboxExample() {
+    const [isChecked, setIsChecked] = React.useState(undefined);
+    return (
+        <Checkbox
+            isIndeterminate={isChecked === undefined}
+            isChecked={isChecked}
+            onChange={setIsChecked}
+        >
+            Select all cities
+        </Checkbox>
+    );
 }
 ```
 
 ## Disabled checkboxes
 
-The `isDisabled` prop visually and functionally disabled the radio button. It also visually disables the related label.
+The `isDisabled` prop visually and functionally disabled the radio button.
```

Comment:
Can we revert the line-length changes? Unrelated to the PR and not consistent with the rest of the MDX files.